components/event/EventCard.js

```diff
@@ -66,11 +66,11 @@ export default function EventCard({ manage, event, usernames }) {
                     </Link>
                   )}
                 </div>
-                {event.userStatus && (
+                {event.isSpeaking && (
                   <div className="text-primary-medium-low dark:text-primary-low-medium italic hidden lg:block">
-                    {event.userStatus}
-                    {event.userStatus == "speaking" && " at "} this event
-                    {event.userStatus == "speaking" && event?.speakingTopic && (
+                    {event.isSpeaking}
+                    {"Speaking at "} this event
+                    {event.isSpeaking  && event?.speakingTopic && (
```

Comment:
The `event.isSpeaking` check is unnecessary as this section is only displayed when this is true.